build/_k-standard-goals.shade

```diff
@@ -118,6 +110,38 @@ default DNX_TOOLS_FEED = 'https://www.myget.org/F/dnxtools/api/v2'
     }
   }
 
+#build-compile target='compile' if='IsBuildV2'
```

Comment:
I know it doesn't really matter, but should this run as part of `test-compile`instead?